src/main/java/ui/components/pickers/PickerMilestone.java

```diff
@@ -23,8 +23,8 @@
     private static final int BIG_LABEL_FONT = 16;
 
     private boolean isSelected = false;
-    private boolean isMatching = false;
     private boolean isExisting = false;
+    private boolean isMatching = true;
```

Comment:
Rather than to be consistent with others, I am thinking more on how the behaviour should be especially that things might change in the future. Setting to `false` makes sense to me because nothing should be matching at the start unless you do some preprocessing (say the bestMatching algorithm).

That being said, if the intended behaviour is for all `PickerMilestone` to match at the start (that is general to any algorithm other than best matching, say #1413), then this is fine.